account_banking_sepa_credit_transfer/models/account_payment_method.py

```diff
@@ -23,7 +23,8 @@ def get_xsd_file_path(self):
         if self.pain_version in [
                 'pain.001.001.02', 'pain.001.001.03', 'pain.001.001.04',
                 'pain.001.001.05', 'pain.001.003.03']:
-            path = 'account_banking_sepa_credit_transfer/data/%s.xsd'\
-                % self.pain_version
+            abs_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__))) 
```

Comment:
This is not the way. There's a method in odoo.tools for getting the correct path.

Reply:
Hi @pedrobaeza , 

humm, I was expecting this but I didn't found this or maybe did I make a wrong interpretation of methods here :
* https://github.com/odoo/odoo/blob/9.0/openerp/tools/appdirs.py 

Actually or deployment is done via multiple paths in addons_path and i'm not sure to find the correct method for getting the module path itself.
Did I searched in a wrong file?

regards

Reply:
The method is `tools.file_open` with the relative path and it returns a class that encloses the opened file. Some other methods `tools.convert_xml_import` directly accept that class.